Formula/solr@8.11.rb

```diff
@@ -0,0 +1,58 @@
+class SolrAT811 < Formula
+  desc "Enterprise search platform from the Apache Lucene project"
+  homepage "https://solr.apache.org/"
+  url "https://dlcdn.apache.org/lucene/solr/8.11.2/solr-8.11.2.tgz"
+  mirror "https://archive.apache.org/dist/lucene/solr/8.11.2/solr-8.11.2.tgz"
+  sha256 "54d6ebd392942f0798a60d50a910e26794b2c344ee97c2d9b50e678a7066d3a6"
+  license "Apache-2.0"
+
+  keg_only :versioned_formula
+
+  depends_on "openjdk"
+
+  def install
+    pkgshare.install "bin/solr.in.sh"
+    (var/"lib/solr").install "server/solr/README.txt", "server/solr/solr.xml", "server/solr/zoo.cfg"
+    prefix.install %w[contrib dist server]
+    libexec.install "bin"
+    bin.install [libexec/"bin/solr", libexec/"bin/post", libexec/"bin/oom_solr.sh"]
+
+    env = Language::Java.overridable_java_home_env
+    env["SOLR_HOME"] = "${SOLR_HOME:-#{var/"lib/solr"}}"
+    env["SOLR_LOGS_DIR"] = "${SOLR_LOGS_DIR:-#{var/"log/solr"}}"
+    env["SOLR_PID_DIR"] = "${SOLR_PID_DIR:-#{var/"run/solr"}}"
+    bin.env_script_all_files libexec, env
+    (libexec/"bin").rmtree
+
+    inreplace libexec/"solr", "/usr/local/share/solr", pkgshare
+  end
+
+  def post_install
+    (var/"run/solr").mkpath
+    (var/"log/solr").mkpath
+  end
+
+  service do
+    run [opt_bin/"solr", "start", "-f", "-s", HOMEBREW_PREFIX/"var/lib/solr"]
+    working_dir HOMEBREW_PREFIX
+  end
+
+  test do
+    ENV["SOLR_PID_DIR"] = testpath
+    port = free_port
+
+    # Info detects no Solr node => exit code 3
+    shell_output(bin/"solr -i", 3)
+    # Start a Solr node => exit code 0
+    shell_output(bin/"solr start -p #{port} -Djava.io.tmpdir=/tmp")
+    # Info detects a Solr node => exit code 0
+    shell_output(bin/"solr -i")
+    # Impossible to start a second Solr node on the same port => exit code 1
+    shell_output(bin/"solr start -p #{port}", 1)
+    # Stop a Solr node => exit code 0
+    # Exit code is 1 in a docker container, see https://github.com/apache/solr/pull/250
```

Comment:
```
  solr@8.11:
    * Formula refers to a GitHub issue or pull request that is closed: https://github.com/apache/solr/pull/250
```
This should be allowed, I think. Especially since the change from the PR was committed to `main` (though the PR is closed). CC @SMillerDev @Homebrew/brew

Reply:
Specifically, following changes in commit (was backported in 9.1.1 so can see error message in that PR):
```sh
# This can happen if, for example, you are running Solr inside a docker container with multiple processes
# rather than running it is as the only service. The --init flag on docker avoids that particular problem.
echo -e "Solr process $SOLR_PID has terminated abnormally. Solr has exited but a zombie process entry remains."
exit 1
```